src/chapters/more-on-functions/functions-as-values.rst

```diff
@@ -0,0 +1,67 @@
+Functions as Values
+===================
+
+.. index:: function; as a value
+
+As we have noted, functions are a powerful programming feature. JavaScript in particular allows functions to be used in flexible ways that are not possible in many other languages. This chapter introduces a bit more of the power of functions.
+
+Functions Are Data
+------------------
+
+We :ref:`defined a value <def-value>` as "a specific piece of data." Some examples are the number ``42``, the string ``"Hello, World!"``, and the array ``["MO", "FL", "DC"]``. *Functions are also values*, and while they appear to be very different from other values we have worked with, they share many core characteristics.
+
+.. index:: data type
+
+In particular, functions have a data type, just like all other values. Recall that a **data type** is a group of values that share characteristics, such as strings and numbers. All strings have a length, while numbers don't. Numbers can be manipulated in ways that strings cannot be, via operations like division and subtraction. 
+
+.. admonition:: Example
+
+   The data type of the type conversion function ``Number`` is ``function``. In fact, all functions are of type ``function``.
+
+   .. sourcecode:: js
+   
+      console.log(typeof Number);   
```

Comment:
Good call. I added statements for a number and a string, but left off an array since it reports its type as `object`. Since we haven't discussed this, it could be confusing.